datadog/resource_datadog_dashboard_test.go

```diff
@@ -702,6 +712,15 @@ func TestAccDatadogDashboard_update(t *testing.T) {
 					resource.TestCheckResourceAttr("datadog_dashboard.ordered_dashboard", "widget.12.group_definition.0.widget.1.alert_graph_definition.0.viz_type", "toplist"),
 					resource.TestCheckResourceAttr("datadog_dashboard.ordered_dashboard", "widget.12.group_definition.0.widget.1.alert_graph_definition.0.title", "Alert Graph"),
 					resource.TestCheckResourceAttr("datadog_dashboard.ordered_dashboard", "widget.12.group_definition.0.widget.1.alert_graph_definition.0.time.live_span", "1h"),
+					// Service Level Objective widget
+					resource.TestCheckResourceAttr("datadog_dashboard.ordered_dashboard", "widget.13.scatterplot_definition.0.title", "Widget Title"),
```

Comment:
In all these steps, you want to look for `service_level_objective_definition`, not `scatterplot_definition`.